src/lib.rs

```diff
@@ -424,6 +427,9 @@ unsafe fn build_font_image(font: rusttype::Font, characters_list: Vec<char>, fon
     // a margin around each character to prevent artifacts
     const MARGIN: u32 = 2;
 
+    // glyph size for characters not presented in font.
+    let invalid_character_width = font_size / 4;
```

Comment:
Changing this line to `let invalid_character_width = font_size / 4;` makes it work well with monospace fonts without making proportional look bad.